pos_edit_order_line/models/pos_config.py

```diff
@@ -0,0 +1,14 @@
+# copyright 2022 Dinar Gabbasov
+# Copyright 2022 Ooops404
+# Copyright 2022 Cetmix
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    allow_edit_order_line = fields.Boolean(
+        default=False,
```

Comment:
Well. When we install a module we expect we want the feature. So set default=True is better IMO.